CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to Archive of Our Own
```

Comment:
Since we're not actually producing a piece of software called "OTW-Archive" but rather a website called "Archive of Our Own", imo, the answer is no. People who want to contribute will be contributing to the website.
Obviously, I'll change it if anyone feels strongly that it should be "OTW-Archive" though!